### DISCUSSION TITLE
Trying to have access to database

### DISCUSSION THREAD

**Discussion by ghoster1234** [learningequality/kolibri#9599] [Q&A] [answered]
I got kolibri to run locally and trying to read the database (for example, using datagrip) but have no lucks on finding the database info. Would anyone be able to show me which folder holds database's name, password, etc? Thanks

**Comment by rtibbles** [comment, marked as answer]
Hi @ghoster1234 - unless you have configured it otherwise, Kolibri will store most of its information in a SQLite database in the `KOLIBRI_HOME` folder. This folder defaults to a `.kolibri` folder inside the operating system user directory (where exactly that is located will depend on your operating system).

Once you have located this folder, it will be in a file called `db.sqlite3`. Database configuration can be traced back from the Django settings that Kolibri uses, see here: https://github.com/learningequality/kolibri/blob/develop/kolibri/deployment/default/settings/base.py#L144
